src/index.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
```

Comment:
It seems like that's not the case with Node 8.  The source was written using native node-8 in mind.  Relying on babel configs to insert `'use strict'` is not a good idea.

```
jhford-work:~ $ cat test.js
a = 3;
jhford-work:~ $ node --version
v8.2.1
jhford-work:~ $ node test.js
jhford-work:~ $ vim test.js
jhford-work:~ $ cat test.js
'use strict';
a = 3;
jhford-work:~ $ node test.js
/Users/jhford/test.js:2
a = 3;
  ^

ReferenceError: a is not defined
    at Object.<anonymous> (/Users/jhford/test.js:2:3)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3
```